### PR TITLE
Fix mobile style.

### PR DESCRIPTION
**Pull request by LautaroCesso**

### CODE REVIEW

File: client/src/app/admin/panel/staff/admin-panel-departments.scss [hunk 1]
```diff
@@ -82,13 +86,14 @@
...
+
                 &__name {
-                    display: unset;
                     margin: 30px;
                 }
 
                 &__private-option {
-                    display: unset;
                     margin: 10px;
                 }
```
Comment (LautaroCesso) [inline client/src/app/admin/panel/staff/admin-panel-departments.scss:98]: Before: 
Screenshot from 2020-06-23 21-52-47

After:
Screenshot from 2020-06-23 21-56-25

File: client/src/app/main/main-layout-header.scss [hunk 2]
```diff
@@ -17,33 +17,21 @@
...
+    &__languages {
+        right: 5px;
+        position: absolute;
+        bottom: 0;
+        margin: 0 6px;
     }
-}
\ No newline at end of file
+
+}
```
Comment (LautaroCesso) [inline client/src/app/main/main-layout-header.scss:37]: Before:
Screenshot from 2020-06-23 22-04-38

After:
Screenshot from 2020-06-23 22-05-04

File: client/src/app/main/main-layout-footer.scss [hunk 3]
```diff
@@ -37,4 +37,17 @@
...
+                &__powered,
+                &__extra-link {
+                    float: unset;
+                    padding: 5px;
+                }
+            }
+        }
+
+    }
+}
```
Comment (LautaroCesso) [inline client/src/app/main/main-layout-footer.scss:52]: Before:
Screenshot from 2020-06-23 22-10-27

After:
Screenshot from 2020-06-23 22-10-39

File: client/src/app/main/main-check-ticket-page.scss [hunk 4]
```diff
@@ -21,4 +21,21 @@
...
+                width: 300px;
+                margin: 0;
+                display: unset;
+            }
+            &__form {
+                width: 280px;
+            }
+        }
+    }
+
```
Comment (LautaroCesso) [inline client/src/app/main/main-check-ticket-page.scss]: Before:
Screenshot from 2020-06-24 14-46-53

After:
Screenshot from 2020-06-24 14-47-06

File: client/src/app/admin/panel/staff/admin-panel-departments.scss [hunk 5]
```diff
@@ -81,7 +78,36 @@
...
+                &__checkbox {
+                    display: flex;
+                    flex-direction: row;
+                    justify-content: space-between;
+                    align-items: center;
+                }
+            }
+        }
     }
+
```
Comment (LautaroCesso) [inline client/src/app/admin/panel/staff/admin-panel-departments.scss:117]: Before:
Screenshot from 2020-06-24 14-52-07

After:
Screenshot from 2020-06-24 14-52-48
Screenshot from 2020-06-24 14-52-52

File: client/src/app/admin/panel/settings/admin-panel-system-preferences.scss [hunk 6]
```diff
@@ -51,4 +51,28 @@
...
+                }
+            }
+        }
+    }
+
+    @media screen and (max-width: 415px) {
+        .admin-panel-system-preferences__container__button {
+            margin: 10px;
+        }
+    }
```
Comment (LautaroCesso) [inline client/src/app/admin/panel/settings/admin-panel-system-preferences.scss:79]: Before:
Screenshot from 2020-06-24 15-48-32

After:
Screenshot from 2020-06-24 15-50-02

Comment (LautaroCesso) [reply client/src/app/admin/panel/settings/admin-panel-system-preferences.scss:79]: Before:
Screenshot from 2020-06-24 15-48-44

After:
Screenshot from 2020-06-24 15-50-10

File: client/src/app/admin/panel/articles/admin-panel-list-articles.scss [hunk 7]
```diff
@@ -3,4 +3,11 @@
...
-}
\ No newline at end of file
+
+    @media screen and (max-width: 415px) {
+        .admin-panel-list-articles__list  {
+            padding: 0;
+        }
+    }
+
+}
```
Comment (LautaroCesso) [inline client/src/app/admin/panel/articles/admin-panel-list-articles.scss:13]: Before:
Screenshot from 2020-06-24 16-08-14
Screenshot from 2020-06-24 16-08-22

After:
Screenshot from 2020-06-24 16-08-53

File: client/src/app/admin/panel/staff/admin-panel-staff-members.scss [hunk 8]
```diff
@@ -26,4 +26,21 @@
...
+            &__button {
+                float: unset;
+                margin: 15px;
+            }
+
+            &__wrapper {
+                height: unset;
+            }
+        }
+    }
```
Comment (LautaroCesso) [inline client/src/app/admin/panel/staff/admin-panel-staff-members.scss:45]: Before:
Screenshot from 2020-06-24 16-35-27

After:
Screenshot from 2020-06-24 16-35-46

File: client/src/app/admin/panel/users/admin-panel-ban-users.scss [hunk 9]
```diff
@@ -33,4 +33,33 @@
...
+                align-items: center;
+            }
+
+            &__un-ban-button {
+                float: unset;
+                margin: 15px 0;
+            }
+        }
+    }
+}
```
Comment (LautaroCesso) [inline client/src/app/admin/panel/users/admin-panel-ban-users.scss:65]: Before:
Screenshot from 2020-06-25 00-03-32

After:
Screenshot from 2020-06-25 00-04-33
Screenshot from 2020-06-25 00-04-47

File: client/src/app/admin/panel/staff/admin-panel-departments.scss [hunk 10]
```diff
@@ -7,17 +7,28 @@
...
     }
 
-    &__name {
-        display:inline-block;margin-right: 101px;
+    &__container{
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        align-items: center;
+        width: 80%;
```
Comment (ivandiazwm) [inline client/src/app/admin/panel/staff/admin-panel-departments.scss:18]: please avoid using percentages for width. I'd prefer a static px value

**Pull request merged**